flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBNativeMetricOptions.java

```diff
@@ -401,11 +414,18 @@ public void enableActualDelayedWriteRate() {
 		this.properties.add(RocksDBProperty.ActualDelayedWriteRate.getRocksDBProperty());
 	}
 
+	/**
+	 * Returns the column family as variable.
+	 */
+	public void enableColumnFamilyAsVariable() {
+		this.properties.add(RocksDBProperty.ColumnFamilyAsVariable.getRocksDBProperty());
```

Comment:
I would add this as a separate flag with a dedicated getter. That way you don't have to modify the assertions in the tests.